rebasehelper/cli.py

```diff
@@ -133,6 +133,18 @@ def run():
                     os.chdir(repo_path)
                 except OSError:
                     raise RebaseHelperError('Could not change directory to the cloned repository')
+
+            macros = {}
+            for macro in config.rpmmacros:
+                try:
+                    macro = macro.strip()
+                    key, value = macro.split(' ', 1)
+                    if not key.startswith('%'):
+                        key = '%{}'.format(key)
```

Comment:
I was going to say this is wrong, but it seems to work with `rpm -D`. RPM guide however says:
> Do not place the leading percent sign, %, on the macro you define with --define.

And I'm not sure about `rpm.addMacro()`. After all, you can't use `%` with `%define` or `%global`, so I would rather stay away from it.

Reply:
Do I understand correctly that we shouldn't prepend the percent sign to the macro? What about cases when the user has already prepended the precent sign? Should we strip it?